tests/test_tpp_backend.py

```diff
@@ -2047,22 +2047,22 @@ def test_quote_against_db():
 
 def test_escape_like_query_fragment():
     assert escape_like_query_fragment("foo") == "foo"
-    assert escape_like_query_fragment("foo%bar_") == "foo\\%bar\\_"
+    assert escape_like_query_fragment("foo%bar_") == "foo!%bar!_"
 
 
 def test_escape_like_query_fragment_against_db():
```

Comment:
Oh, I did test this when I first wrote it! Now I'm very confused as to what was causing the error we saw earlier. But in any case, I'm pretty sure this change will fix it because we don't need to worry about double-escaping.

Reply:
I think I get why the tests passed because in the context of a Python string `\\` is how you spell `\` (i.e. `print("\\")` outputs `\`) but I don't understand how the double blackslash then made its way into SQL and triggered that error. Anyway, thanks for fixing!